video/out/gpu/context.c

```diff
@@ -49,6 +49,7 @@ extern const struct ra_ctx_fns ra_ctx_android;
 extern const struct ra_ctx_fns ra_ctx_vulkan_wayland;
 extern const struct ra_ctx_fns ra_ctx_vulkan_win;
 extern const struct ra_ctx_fns ra_ctx_vulkan_xlib;
+extern const struct ra_ctx_fnd ra_ctx_vulkan_android;
```

Comment:
typo here `fnd` instead of `fns`

Reply:
https://github.com/mpv-player/mpv/commit/afe2b83183aa549ba45e9228a48a36f6d3cf4a6a